thirdparty_containers/tomcat/playlist.xml

```diff
@@ -15,7 +15,9 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>tomcat_test</testCaseName>
-	<command>docker run --rm $(EXTRA_DOCKER_ARGS) adoptopenjdk-tomcat-test:latest ; \
+	<command>docker run --name tomcat-test $(EXTRA_DOCKER_ARGS) adoptopenjdk-tomcat-test:latest ; \
+		docker cp tomcat-test:/tomcat/output/build/logs $(REPORTDIR)/external_test_reports; \
```

Comment:
I would suggest moving `docker cp and docker rm` after `${TEST_STATUS}` like other tests. Otherwise the ${TEST_STATUS} will be determined by the status of `docker rm`.

Reply:
Also if tests passed ${TEST_STATUS} will clean the folder $(REPORTDIR). That means all copied files will be deleted. Nothing can be consumed by junit plugin.

Reply:
Thanks Sophia have made the change. Please review.